### PR TITLE
[fix] New theme upload

### PR DESCRIPTION
**Pull request by Its-Just-Nans**
[fix] add toastService to directive
[fix] change place to upload
[feature] add modal open

### CODE REVIEW

File: backoffice/js/directives.js [hunk 1]
```diff
@@ -2126,8 +2126,8 @@ adminCatagenDirectives.directive("nsUploadFiles", [
             },
             templateUrl: "views/templates/nsUploadFiles.html",
             controller: [
-                "$scope", "Upload",
-                function ($scope, Upload)
+                "$scope", "Upload", "toastService",
```
Comment (Its-Just-Nans) [inline backoffice/js/directives.js:2129]: At line 2300 this directive use toastService

File: backoffice/app/themes/themes.controllers.js [hunk 2]
```diff
@@ -281,4 +278,31 @@ ThemesController.controller("ThemesCtrl", [
...
+            toastService.toast("success", "Thème ajouté ! Pour l'utiliser, il suffit de le selectionner");
+            $modalInstance.close("save");
+        };
+
+
+        $scope.cancel = function (){
+            $modalInstance.dismiss("cancel");
+        };
+    }
+]);
```
Comment (Its-Just-Nans) [inline backoffice/app/themes/themes.controllers.js:308]: Controller of the modal

File: backoffice/app/themes/themes.controllers.js [hunk 3]
```diff
@@ -281,4 +278,31 @@ ThemesController.controller("ThemesCtrl", [
             };
 
     }
-]);
\ No newline at end of file
+]);
+
+CategoryControllers.controller("ThemesNewCtrl", [
```
Comment (devaquila) [inline backoffice/app/themes/themes.controllers.js:283]: CategoryControllers ?!

Comment (Its-Just-Nans) [reply backoffice/app/themes/themes.controllers.js]: Fixed

**Pull request merged**